src/api/component.js

```diff
@@ -219,6 +209,9 @@ export default class extends HTMLElement {
     // Ensures the component can't be rendered while disconnected.
     this[$connected] = false;
 
+    // Suspend updating Attributes until re-connected
+    getAttrMgr(this).syncAttributes(false);
```

Comment:
I'd prefer a different method name here. Sort of a nit, but I feel this is a potential [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).